x = 100

try:
    x/0
except Exception as e:
    print(e)